### ISSUE TITLE
loss 不下降

### ISSUE THREAD

**Issue by AndrewJSong** [zhechen/PLARD#6]
感谢你的开源，最近在试着训练你的网络，发现loss并不下降，从1左右迅速下降到0.5之后（不到一个epoch），就不再下降，而且训练出来的权重，测试结果完全没法看，铁别差。环境和你的一样，请问这大概是什么原因呢？谢谢！

**Comment by zhechen** [comment]
please try smaller learning rate

**Comment by AndrewJSong** [comment]
Thank you verymuch!I will try it!

**Comment by zeng32** [comment]
> Thank you verymuch!I will try it!

Same problem with me, Is it working now with smaller LR?

**Comment by AndrewJSong** [comment]
> > Thank you verymuch!I will try it!
> 
> Same problem with me, Is it working now with smaller LR?
我显存快爆了，放弃了。你可以提高batchsize 试一试

**Comment by go925315** [comment]
@AndrewJSong 你的問題有解決嗎？
我測試batchsize 提高到13，l_rate_step 20，loss最小到0.46就不在下降了

**Comment by yingtao123** [comment]
> @AndrewJSong 你的問題有解決嗎？
> 我測試batchsize 提高到13，l_rate_step 20，loss最小到0.46就不在下降了

我也是，loss0.4多就不降了，请问你有解决这个问题吗

**Comment by OSU-Haolin** [comment]
Does someone solve this problem? 
meet the same problem.

**Comment by mei123hao** [comment]
Same problems，It seems that the model can not be trained sucessfully! I also stuck the loss around 0.5...

**Comment by hasirk** [comment]
I am facing the same problem, loss is changing around 0.45
